content/yaml-basic-configuration/yaml-getting-started.md

```diff
@@ -175,15 +175,15 @@ The main sections in each workflow are described below.
 `instance_type:` specifies the [build machine type](../specs/machine-type) to use for the build. The supported build machines are:
 | **Instance Type** | **Build Machine** |
 | ------------- | -----------------  |
-| `mac_mini_m1`    | macOS M1 standard VM (default) |
-| `mac_pro`     | macOS Intel VM            |
-| `linux`       | Linux standard VM           |
-| `linux_x2`    | Linux premium VM            |
-| `windows_x2`  | Windows premium VM          |
+| `mac_mini_m1`    | macOS M1  VM (default)   |
+| `mac_mini_m2`    | macOS M2  VM             |
+| `mac_pro`     | macOS Intel VM              |
+| `linux`       | Linux         VM            |
```

Comment:
This should still be `linux_x2`, or we should list both.

Reply:
I disagree. Wouldn't `linux_x2` raise concerns since on pricing page we only have Linux instance type and on invoicing we also charge for Linux.